### DISCUSSION TITLE
啥时候能支持字符串类型的字段嘞？

### DISCUSSION THREAD

**Discussion by allforyou2016** [milvus-io/milvus#10087] [Q&A and General discussion] [answered]
啥时候能支持字符串类型的字段嘞？想在获取检索结果集的同时返回其他一些信息

**Comment by yhmo** [comment, marked as answer]
The work for string type field has begun two month ago, the [design doc](https://wiki.lfaidata.foundation/display/MIL/MEP+11+--+Support+String+DataType) is ready, but was pending because we got too many stability issues. 
This feature cannot be delivered in 2.0GA, we will resume the work and deliver it in v2.1

**Reply by sportzhang** [reply]
希望尽快支持string类型！！！
